modules/admin/pom.xml

```diff
@@ -363,4 +359,46 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>FT</id>
```

Comment:
This profile should also handle the copying and cleaning of the configuration on its own.